puzzles/is-palindrome/isPalindrome.test.js

```diff
@@ -0,0 +1,33 @@
+const isPalindrome = require('./isPalindrome');
```

Comment:
A test should also be provided for each puzzle. (to make sure the solution is correct, and all edge cases are covered).